test/helpers/test_provisioner.rb

```diff
@@ -68,6 +68,17 @@ def deploy_metric_server
         found = kubectl.run("get", "-f", resource, log_failure: false).last.success?
         kubectl.run("create", "-f", resource) unless found
       end
+
+      # The service is often slow to start leading to randomly failed HPA tests
+      3.times do |round|
+        service = kubectl.run("-n", "kube-system", "get", "APIService", "v1beta1.metrics.k8s.io", "-o", "json")
```

Comment:
Is there a reason not to use the `attempts` parameter for `kubectl.run` here?

Reply:
I expect the resource to have been created by line 69 so probably being a bit more defensive than needed, but its being ready that I would normally expect to fail and that attempts wont help with